common/app/layout/FaciaContainer.scala

```diff
@@ -165,7 +166,9 @@ object FaciaContainer {
         case _            => ContainerCommercialOptions(omitMPU = false, adFree = adFree)
       },
       config.config.description.map(DescriptionMetaHeader),
-      None,
+      customClasses = config.config.metadata
+        .flatMap(paletteClass(container, _))
+        .map(Seq(_, "fc-container--has-palette")),
```

Comment:
Did you think about whether this `--has-palette` logic class could live in `GetClasses` instead?

Reply:
Yes, I think that makes more sense, pushing the change up now